### ISSUE TITLE
[Visual Requirements - Make Code - Import File] Luminosity ratio is less than 4.5:1 for text control present on 'Import File' pop-up.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3566]
**User Experience:**
Visual impaired user would find difficulty to see the control or element if that controls have insufficient color contrast as they won't be able to take on that controls and he/she would miss the functionality of such control or element.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to "Import" button and select it.
4. Navigate through the controls appearing under "Import" dialog.
5. Navigate to import file option and navigate through the all the controls of dialog.
6. Verify contrast ratio of text control(You can import files by dragging and dropping them anywhere in the editor!)

**Actual Result:**
Luminosity ratio is less than 4.5:1 i.e. 4.3:1 for text control(You can import files by dragging and dropping them anywhere in the editor!) under 'Import File' pop-up.

**Expected Result:**
Luminosity ratio should be more than and equal to 4.5:1 for text control(You can import files by dragging and dropping them anywhere in the editor!) under 'Import File' pop-up.

**Note:**
Same issue exists on Firefox & Chrome.

**WCAG Reference:**
[WCAG 1.4.3 - Non-text Contrast
](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Tool: CCA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG1.4.3, #Win10-Edge(Chromium)

![WCAG1 4 3_Import File_CCA](https://user-images.githubusercontent.com/72871089/97963974-004e2a80-1dde-11eb-87fd-5efb4e8fac48.jpg)
